File: Python/Program/ListBarang.py
class ListBarang:
    def __init__(self, id: int = 0, namaBarang: str = "", harga: float = 0.0):
        self.id = id
        self.namaBarang = namaBarang
        self.harga = harga

    # Getter
    def getId(self): return self.id
    def getNamaBarang(self): return self.namaBarang
    def getHarga(self): return self.harga

    # Setter
    def setId(self, id): self.id = id
    def setNamaBarang(self, nama): self.namaBarang = nama
    def setHarga(self, harga): self.harga = harga

File: Java/Program/ListBarang.java
public class ListBarang {
    private int id;
    private String namaBarang;
    private double harga;

    public ListBarang(int id, String namaBarang, double harga) {
        this.id = id;
        this.namaBarang = namaBarang;
        this.harga = harga;
    }

    public int getId() { return id; }
    public void setId(int id) { this.id = id; }

    public String getNamaBarang() { return namaBarang; }
    public void setNamaBarang(String namaBarang) { this.namaBarang = namaBarang; }

    public double getHarga() { return harga; }
    public void setHarga(double harga) { this.harga = harga; }
}
